nexus/db-model/src/schema.rs

```diff
@@ -468,6 +468,46 @@ table! {
     }
 }
 
+table! {
+    affinity_group (id) {
+        id -> Uuid,
+        name -> Text,
+        description -> Text,
+        time_created -> Timestamptz,
+        time_modified -> Timestamptz,
+        time_deleted -> Nullable<Timestamptz>,
+        policy -> crate::AffinityPolicyEnum,
+        distance -> crate::AffinityDistanceEnum,
+    }
+}
+
+table! {
+    anti_affinity_group (id) {
+        id -> Uuid,
+        name -> Text,
+        description -> Text,
+        time_created -> Timestamptz,
+        time_modified -> Timestamptz,
+        time_deleted -> Nullable<Timestamptz>,
+        policy -> crate::AffinityPolicyEnum,
+        distance -> crate::AffinityDistanceEnum,
+    }
+}
+
+table! {
+    affinity_group_instance_membership (group_id, instance_id) {
+        group_id -> Uuid,
+        instance_id -> Uuid,
+    }
+}
+
+table! {
+    anti_affinity_group_instance_membership (group_id, instance_id) {
+        group_id -> Uuid,
+        instance_id -> Uuid,
+    }
+}
```

Comment:
I presume we'll want to add `joinable!`/`allow_tables_to_appear_in_same_query!` for joining the affinity/anti-affinity group tables on the instances table?

Reply:
Yeah, I usually add these lazily when rustc complains at me

Reply:
Reasonable!